pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1275 @@
+
+**Important note:**
+
+In order for `TrilioVault` to work correctly and to backup your `PVCs`, `OVH Managed Kubernetes Cluster` needs to be configured to support the `Container Storage Interface` (or `CSI`, for short). By default it comes with the driver already installed and configured. You can check using below command:
+
+```shell
+kubectl get storageclass
+```
+
+The output should look similar to (notice the provisioner is [hostpath.csi.k8s.io](https://github.com/kubernetes-csi/csi-driver-host-path)):
+
+```text
+NAME                        PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+csi-cinder-classic          cinder.csi.openstack.org   Delete          Immediate           true                   3d
+csi-cinder-high-speed       cinder.csi.openstack.org   Delete          Immediate           true                   3d
+csi-hostpath-sc (default)   hostpath.csi.k8s.io        Retain          Immediate           false                  2d
+```
+
+## Step 1 - Installing TrilioVault for Kubernetes
+
+In this step, you will learn how to deploy `TrilioVault` for `OVH Managed Kubernetes Cluster`, and manage `TVK` installations via `Helm`. Backups data will be stored in the `OVH Object Storage` bucket created earlier in the [Prerequisites](#prerequisites) section.
+
+`TrilioVault` application can be installed as below:
+
+- The `TrilioVault Operator` (installable via `Helm`). You define a `TrilioVaultManager` CRD, which tells `TrilioVault` operator how to handle the `installation`, `post-configuration` steps, and future `upgrades` of the `Trilio` application components.
+- The `TrilioVault Manager` via the [k8s-triliovault] yaml definition (https://docs.trilio.io/kubernetes/use-triliovault/installing-triliovault#upstream-kubernetes)
+	
+
+### Installing TrilioVault Operator and Manager Using Helm
+
+**Important note**:
+
+`OVH Docs` tutorial is using the `Cluster` installation type for the `TVK` application (`applicationScope` Helm value is set to `"Cluster"`). All examples from this tutorial rely on this type of installation to function properly.
+
+Please follow the steps below, to install `TrilioVault` via `Helm`:
+
+1. First, clone the `OVH Docs` Git repository and change directory to your local copy:
+
+    ```shell
+    git clone https://github.com/ovh/docs.git
+
+    cd docs/pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/
+    ```
+
+2. Next, add the `TrilioVault` Helm repository, and list the available charts:
+
+    ```shell
+    helm repo add triliovault http://charts.k8strilio.net/trilio-stable/k8s-triliovault
+
+    helm search repo triliovault
+    ```
+
+    The output looks similar to the following:
+
+    ```text
+    NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
+    triliovault-operator/k8s-triliovault-operator   2.6.4           2.6.4           K8s-TrilioVault-Operator is an operator designe...
```

Comment:
Not the same version:
"triliovault/k8s-triliovault-operator	0.9.0        	0.9.0      	K8s-TrilioVault-Operator is an operator designe..."

Reply:
@raclepoulpe Thank you for the review, version `0.9.0` is deprecated we will be removing it from the repo soon. The user will be using the `2.6.4` version for the `triliovault-operator` installation. I have updated the output of the command as per your suggestion.